tests/main.tf

```diff
@@ -1,20 +1,36 @@
+#terraform {
+#    backend "s3" {
+#      bucket         = "glueops-terraform-eks-backend"
+#      key            = "terraform.tfstate"
+#      region         = "us-west-2"
+#      dynamodb_table = "glueops-terraform-eks-backend"
+#  }
+#}
+
 module "captain" {
-  iam_role_to_assume = "arn:aws:iam::761182885829:role/glueops-captain"
```

Comment:
This account number is dedicated for tests. Where as the account we gave you was for personal dev work